SEP-0000.md

```diff
@@ -0,0 +1,29 @@
+# SEP-0000 -- Index of SunPy Enhancement Proposals (SEPs)
+
+| SEP           | 0 |
```

Comment:
I agree with @ayshih on the heading look here...  but that's the format used by [peps](https://www.python.org/dev/peps/).  Maybe a last modified field will make it easier to find out if something has changed more than trying to find it in the git logs.

Also, I can foresee a problem on how to do this machine readable...  PEPs uses plain text and some of the values are generated automatically for the we website.  If we don't change the order of the tables I imagine we could write a parser, so in the future we could measure the time of discussion and the authors and other properties...